lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -146,29 +144,20 @@ ban_unavailable_allowed_nodes(pe_resource_t *instance, int max_per_node)
  * \param[in]     prefer        If not NULL, attempt early assignment to this
  *                              node, if still the best choice; otherwise,
  *                              perform final assignment
- * \param[in]     all_coloc     If true (indicating that there are more
- *                              available nodes than instances), add all parent
- *                              colocations to instance, otherwise add only
- *                              negative (and for "this with" colocations,
- *                              infinite) colocations to avoid needless
- *                              shuffling of instances among nodes
  * \param[in]     max_per_node  Assign at most this many instances to one node
  *
  * \return true if \p instance could be assigned to a node, otherwise false
  */
 static bool
 assign_instance(pe_resource_t *instance, const pe_node_t *prefer,
-                bool all_coloc, int max_per_node)
+                int max_per_node)
 {
     pe_node_t *chosen = NULL;
     pe_node_t *allowed = NULL;
 
     CRM_ASSERT(instance != NULL);
-    pe_rsc_trace(instance,
-                 "Assigning %s (preferring %s, using %s parent colocations)",
```

Comment:
I don't think so; we log a message in pcmk__assign_instances() once per collective rather than for each instance. It doesn't contain that exact info but it can be inferred.